ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.16-nightly-x86.yaml

```diff
@@ -44,21 +45,20 @@ tests:
       COMPUTE_NODE_REPLICAS: "3"
       COMPUTE_NODE_TYPE: Standard_D4s_v5
       CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v5
-      CUSTOM_AZURE_REGION: centralus
       PROFILE_TYPE: reporting
     test:
     - chain: openshift-qe-control-plane
-    workflow: openshift-qe-installer-azure-ipi-ovn-ipsec
+    workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
 - always_run: false
   as: data-path-ipsec-9nodes
   steps:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      COMPUTE_NODE_TYPE: Standard_D8s_v3
+      COMPUTE_NODE_TYPE: Standard_D8s_v5
       CUSTOM_AZURE_REGION: centralus
-      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_D16s_v3
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_D16s_v5
```

Comment:
Update Infra nodes to `Standard_E4s_v5`
Reference: [URL](https://docs.google.com/presentation/d/1OMtiM6S0yYCcE-sUeSNYfbSyZe8EtjQ2FGu-hqLdul4/edit#slide=id.gc52980a47d_1_0)

Reply:
@krishvoor According to @rsevilla87, the ingress-perf test needs larger 16vCPU instance type because ingress component running on it. In AWS we are using [c5.4xlarge](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml#L159C7-L159C53) (16 vCPUs 64 GiB). I think we should update this to this doc https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit#heading=h.bqtllo1nrbil

Reply:
Yes, depends on the ingress-perf configuration used (https://github.com/cloud-bulldozer/e2e-benchmarking/tree/master/workloads/ingress-perf/config), self-Managed OCP has a higher workload, I used Standard_D16s_v5 referencing the AWS configuration for the same test.